### ISSUE TITLE
[Bug] TabView does not switch tabs on UWP

### ISSUE THREAD

**Issue by mgierlasinski** [xamarin/XamarinCommunityToolkit#887]
### Description
Tapping on other tab does not switch current tab. Looks like indicator is trying to move into other tab but it goes back, `CarouselView` tries to move content but goes back too.

### Steps to Reproduce

1. Open Xamarin Community Toolkit sample app.
2. Go to Views -> TabView -> Getting Started.
3. Tap on `Tab 2`.

### Expected Behavior
`TabView` is switching tab.

### Actual Behavior
`Tab 1` remains visible.

### Basic Information

- Version with issue: build from sources on develop branch, 1.0.2 is current stable release.
- Last known good version: unknown
- IDE: Visual Studio 2019 Enterprise 16.8.5
- Platform Target Frameworks: 
  - UWP:  18362
- Nuget Packages: referenced by XCT Sample app on develop branch

### Workaround
Double tap on other tab to switch.

### Reproduction imagery

![TabView](https://user-images.githubusercontent.com/19707868/107660679-d0d67600-6c88-11eb-85d7-87cc4603416a.gif)

**Comment by pooran** [comment]
Any update on this?

**Comment by ninaada** [comment]
We are also facing the same issue. The tab just does not switch.

**Comment by Orgbrat** [comment]
So folks this is a show stopper on UWP Shell application. The tab just does not switch no matter what you do. Double tapping tab does not make it switch.  And I am using the very latest XCT v1.2.0-pre2.

**Comment by techravi007** [comment]
Is there any workaround until it get fixed in next coming release.
desperately looking for an update on it.

**Comment by Orgbrat** [comment]
@techravi007 Have you heard or read something that definitely states this issue will be fixed in the next coming release?

**Comment by techravi007** [comment]
@Orgbrat No, I haven't any clue if it is going to be fixed in the next coming release though i'm hoping they should fix it.

**Comment by Orgbrat** [comment]
Well I have never done anything with someone's open source project before, but I have forked the Xamarin.Community.Toolkit repository and I am going to see if I can tell what is going on with the TabView not working correctly in UWP. Not making any promises but at least I do have everything I need to work on UWP on my machine although I am not a UWP expert. But I am willing to give it a shot.  Worst thing can happen is that I fail to help the issue...  :)

**Comment by Agredo** [comment]
@Orgbrat Thank you! If You need help I hope I can help. We need it too working for UWP.

**Comment by Orgbrat** [comment]
@Agredo  Well unfortunately I have received a new project to work on and my time is very limited for this issue. I will get back to this as soon as I can. Sorry...

**Comment by themarwik** [comment]
this is still a big issue

**Comment by FrozDark** [comment]
It seems to be bug of CarouselView as it is placed inside TabView for transition switch tabs

**Comment by joseluisct** [comment]
Same problem here. Many times the tab that is shown when you click the strip is a wrong or previous tab
